decidim-bulletin_board-app/app/models/concerns/message.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Message
```

Comment:
Not a fan of this name for the concern but I also don't like the `ActsAsStuff` thing. What about something like `IdentifiedByMessageId` or `HasMessageIdentifier` ?

Reply:
Yes, I don't like it neither. I used this name because the following reasons, but I'd definitely like to find a longer and more specific name:
* This concern is meant to be used in all the models that represent a message (LogEntry and PendingMessage). 
* We already have a `HasMessageIdentifier` concern in the `commmands` folder (and it's slightly different from this one, so we can't reuse it here).

Reply:
I only can think about `ActsAsMessage`. I blame the ruby community for this! XD